pkgs/development/python-modules/kaitaistruct/default.nix

```diff
@@ -1,5 +1,19 @@
-{ lib, buildPythonPackage, fetchPypi }:
+{ lib
+, buildPythonPackage
+, fetchPypi
+, fetchFromGitHub
+, withCompress ? false
```

Comment:
Maybe we should enable that by default.

Reply:
Agreed. I don't even think we need to make this an option.